js/app.jsx

```diff
@@ -297,101 +416,94 @@ var CaseverListItem = React.createClass({
   }
 });
 
+var SearchTableHeadConstTh = React.createClass({
+  render: function(){
+    return(
+      <th>{this.props.text}</th>
+    )
+  }
+})
 
-var CaseverList = React.createClass({
+var SearchTableHeadTh = React.createClass({
```

Comment:
Consider dropping the "Search" in the name. See the comment on `var SearchTable = ...`